packages/forms/src/UIForm/fields/TimezoneList/TimezoneList.component.js

```diff
@@ -0,0 +1,32 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import Datalist from '../Datalist';
+
+import { getTimezones } from './TimezoneList.utils';
+
+function TimezoneList(props) {
+	const { schema } = props;
+	const { lang, schema: { type } } = schema;
```

Comment:
Maybe there's a way to get lang from parent `<Form />` component 💭

Reply:
No "lang"/"language" prop accepted by UI's `<Form />`, `t` not passed to forms widgets, assuming this is not possible atm